css/css-scroll-snap/scroll-target-margin-002.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and scroll-margin with snapping off (y-axis)</title>
```

Comment:
This test does not use focus(), it uses scrollIntoView(). You should fix the title accordingly.

Reply:
Fixed.